src/scaict_uwu/core/config/config.py

```
@@ -0,0 +1,17 @@
+Config
+"""
+
+# Standard imports
+import sys
+from typing import ClassVar
+
+
+class Config:
+    options: ClassVar[dict]
+
+    def get_option(self, name: str):
```

Ruff (0.8.0)

Missing return type annotation for public function `get_option`

(ANN201)